core/unit_test/tstNeighborList.hpp

```diff
@@ -24,6 +24,7 @@ namespace Test
 {
 //---------------------------------------------------------------------------//
 // Linked cell list cell stencil test.
+/*
```

Comment:
I commented out the portion of the neighbor list test that checks the `linkedCellStencil` since that has been moved into the linked cell list directly. We should probably move the contents of this test into the `linkedCellList` test and alter it to use the stencil as part of the linked cell list

Reply:
I agree, let's move this test